setup.py

```diff
@@ -15,7 +15,11 @@
 import glob
 import platform
 import subprocess
-from distutils import sysconfig
+
+try:
+    from distutils import sysconfig
+except ImportError:
+    import sysconfig
```

Comment:
Looks like this package was introduced in Python 3.2, can we just use it always?

Reply:
https://github.com/PennyLaneAI/catalyst/pull/653/commits/fc46723b98bfcca10f294b01a5efead352a962c1